pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,19 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	bcs := []string{}
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(tagNode, buildedges.BuildOutputEdgeKind) {
+		bcs = append(bcs, f.ResourceName(bcNode))
+	}
+	if len(bcs) > 0 {
+		return osgraph.Suggestion(fmt.Sprintf("%s needs to be imported or created by a build(s) of %s.", f.ResourceName(tagNode), strings.Join(bcs[:], ",")))
```

Comment:
If there is only one build config, we can suggest running `oc start-build`. I would prefer if we separated these cases: no bcs, one bc, multiple bcs.